### PR TITLE
Feat/real data martin

### LINKED ISSUE #37: [PBI] Add real dataset
This dataset will process early april -> project will be paused.

### LINKED ISSUE #51: Welcome frame (app intro popup)
### Is your feature request related to a problem? Please describe.

_No response_

### Describe the solution you would like.

**Bienvenue sur *Swiss Proximity***

*Swiss Proximity* est un outil de planification et d’aide à la décision pour penser de manière plus durable nos territoires urbains de demain. Cette interface a été développée par des chercheurs de l'EPFL pour aider les usagers, praticiens, collectivités et décideurs à mieux comprendre leurs territoires du point de vue des "proximités urbaines". 

**Les *proximités urbaines*, c’est quoi?**

Nous considérons qu’un territoire durable devrait non seulement tendre vers la neutralité carbone, mais aussi favoriser l’inclusivité et le progrès social. Ainsi, les proximités urbaines doivent être pensées en terme de:

- densité (nombre de services, magasins, guichets, …) pour réduire les distances de déplacement et les rendre accessibles en modes doux;
- diversité (diversité de services, magasins, guichets, …) pour offrir une variété de choix dans une distance raisonnable;
- nature et convivialité pour favoriser l’hospitalité urbaine et le vivre-ensemble.

Ainsi, nous proposons un outils qui permette aux utilisateur.ice.s de visualiser les proximités urbaines dans les 80 agglomérations suisses. 

********************************************************Un outil d’aide à la planification********************************************************

L’interface s’organise en un outil comparatif comportant deux volets principaux: 

1. L’offre territoriale de proximité
Nous avons catégorisé 10 types de lieu, services, ou infrastructures qui composent nos territoires urbains: apprendre, se cultiver, se soigner, s’aérer, faire du sport, bien manger, faire ses courses, faire les magasins, guichets, transports publics.

2. La demande activité-déplacement de proximité

Permet de visualiser où les gens se déplacent dans la proximité. En regard de l’offre, l’étude de la demande permet d’identifier des poches territoriales où territoire et usages sont discordants. La demande de mobilité est basée sur le modèle voyageur développé par l’Office Fédérale du Développement Territoriale (ARE).

Un troisième volet permet de calculer les isochrones d’accessibilité avec différents mode de transport à travers les agglomérations suisses.

Navigation sur l’interface
L’interface est optimisée pour un usage sur ordinateur, et n’est pas adapté aux smartphones.

La colonne de gauche vous permet de jouer avec les paramètres de la demande et de l’offre de proximité. Vous trouverez plus d’informations, notamment sur la méthode et les données, dans les *infos* en haut à droite.

**Commencer >**

### Describe alternatives you've considered.

_No response_

### Additional context.

_No response_

### LINKED ISSUE #52: [FEATURE] Info page
## Documentation

(More coming soon)

### Proximity criteria formula

The user can interact in three manners with the values projected on the map:

1. Select a series of criteria $x \in \mathcal{X} \subset \{ 'Public','Care','Culture', ... ,'Transport'\}$
2. Specify the criteria-specific diversity index $i_x \in I = \{1, 2, 3, 4, 5 \}$ 
3. Specify a criteria-specific weight $k_x \enspace | \enspace 0 \leq k_x \leq 1$ associated to each criteria

The formula to compute the modified-value Distance $\hat{\delta}$ that will be projected on the map goes as follows:

$$
\hat{\delta}_i = max(\{k_x \cdot Dist_i(x) : x\in\mathcal{X}\}) \enspace \leq 7000 \enspace \forall(x, k, i)
$$

with distances $\hat{\delta}$ and $Dist_i(x)$ in meters.

**Let us take an example.** Let’s say we are interested in defining “proximity” as easy access to the Cultural scene, Sport amenities, and Outdoor locations. When the user selects a series of criteria in the sidebar of the interface, they put together under the hoods $\mathcal{X} = \{'Culture','Sport','Outdoor'\}$. Users can then select, for each criteria the diversity index e.g., $i_{Culture} = 5, \enspace i_{Sport} = 2, \enspace i_{Outdoor} = 2$. This means that the Swiss Proximity engine will look for the distance to the 5 nearest Culture locations, the 2 nearest Sport locations, and the 2 nearest Outdoor location. If a criteria-specific weight is entered e.g., $k_{Culture} = .75$ then the distance will be artificially reduced to give less importance to the Cultural scene in the user’s own definition of the “proximity”.

### Color code

For each zone (Traffic zone or hexagon), the distance value of $\hat{\delta}$ takes a specific color based on the following discretization.

Demand:

| Travel time (up to) | Approx. Distance [m] | Color [Hex] | Cat |
| --- | --- | --- | --- |
| 15’ (walk) | 1200 | #005A32 | A |
| 20’ (walk) | 1600 | #238443 | B |
| 30’ (walk) | 2400 | #41AB5D | C |
| 15’ (bicycle) | 3700 | #78C679 | D |
| 20’ (bicycle) | 5000 | #ADDD8E | E |
| 30’ (bicycle) | 7000 | #D9F0A3 | F |
| > 30’ (bicycle) | > 7000 | #FFFFCC | G |

Supply:

| Share of proximity trips | Color [Hex] |  |
| --- | --- | --- |
| 0 | No fill | G |
| Quantile 1/6 | #D9F0A3 | F |
| Quantile 2/6 | #ADDD8E | E |
| Quantile 3/6 | #78C679 | D |
| Quantile 4/6 | #41AB5D | C |
| Quantile 5/6 | #238443 | B |
| Quantile 6/6 | #005A32 | A |

### Proximity criteria details

| UI | BACKEND | OSM tags | UI infos |
| --- | --- | --- | --- |
| Guichets | Public | community_centre, police, post_box, post_office, bank | Postes, banques, police, … |
| Se soigner | Care | nursing_home, pharmacy, hospital, clinic, doctors, dentist, optician, hairdresser, beauty | Pharmacies, hôpitaux, dentistes, opticiens, coiffeurs, beauté, … |
| Se cultiver | Culture | arts_centre, library, theatre, nightclub, cinema, museum, events_venue, religion (muslim, jewish, christian) | Bibliothèques, théâtres, cinémas, lieux de culte, … |
| S’aérer | Outdoor | park, playground, square, landuse_recreation_ground, water, fountain, riverbank, river | Parcs, places, fontaines, rives, … |
| Bien manger | Catering | restaurant, fast_food, cafe, pub, bar, food_court, biergarten | Restaurants, cafés, pubs, … |
| Faire du sport | Sport | sports_centre, pitch, swimming_pool, swimming_pool, swimming, water_park, fitness_centre, ice_rink, tennis, golf_course, stadium | Piscines, fitness, stades, patinoires, tennis, … |
| Faire ses courses | Provision | marketplace, supermarket, bakery, general, butcher, greengrocery | Super-marchés, boulangeries, marchés, … |
| Faire les magasins | Shopping | kiosk, mall, department_store, convenience, clothes, florist, chemist, books, shoes, furniture | Kiosks, fleuristes, librairies, bricolage, … |
| Apprendre | Education | university, school, college, kindergarten | Ecoles primaires, secondaires, universités, garderies, … |
| Transports publics | Transport | bus_station, taxi, railway (station, alt), tram_stop, bus_stop | Gares, arrêts de bus et tram, zone taxi, … |

### Crédits

Support technique et développement – ENAC IT4R, EPFL

Né au – Laboratoire de Sociologie Urbaine, EPFL

Elaboré à – [Située.ch](http://xn--situe-esa.ch/) 

### Disclaimer

Ce projet utilise des open data, notamment les données cartographiques Open Street Map (OSM), une base de donnée géoréférencée participative (même fonctionnement que wikipédia). Toutes les informations présentes dans l’offre territorial de proximité ne sont pas des données approuvées par une autorité suisse. 

Par ailleurs, les requêtes isochrones font appel à un tiers: XXX.

Cette interface est une démo et invite à des collaborations avec des acteurs locaux pour affiner l’outil. Nous appelons les utilisateurs à signaler d’éventuelles erreurs à l’adresse suivante: marc-edouard.schultheiss@epfl.ch.

### Contact

Marc-Edouard Schultheiss | ing. civil dipl. EPF, Dr. en Sociologie Urbaine

marc-edouard.schultheiss@epfl.ch

### LINKED ISSUE #53: Mock Up of the app
### Is your feature request related to a problem? Please describe.

_No response_

### Describe the solution you would like.

Please find below an updated mockup compatible with the data I uploaded in another comment.

[mockup.pdf](https://github.com/EPFL-ENAC/LASUR-swiss-proximity/files/11856946/mockup.pdf)

### Describe alternatives you've considered.

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by pierreleripoll**
# Describe your changes

Real dataset needed changes in data viewing (properties, access etc) + changes in app UI, new legend feature, variables are selected differently according if demand or supply
See #53 for app mock up reference

# Related GitHub issues and pull requests

- resolve #53 
- resolve #52 
- resolve #51 
- resolve #37

### CODE REVIEW

File: frontend/src/components/InfoMarkdown.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+### Proximity criteria formula
```
Comment (ymarcon) [inline frontend/src/components/InfoMarkdown.md:2]: La webapp est en FR ou EN ? il me semble qu'il faudrait choisir.

File: frontend/src/components/InfoDialog.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <info-markdown />
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import InfoMarkdown from "@/components/InfoMarkdown.md";
+</script>
+
+<style scoped>
+::v-deep.markdown-body p {
```
Comment (ymarcon) [inline frontend/src/components/InfoDialog.vue:12]: There is a warning:

```
[@vue/compiler-sfc] ::v-deep usage as a combinator has been deprecated. Use :deep(<inner-selector>) instead.
```

See https://stackoverflow.com/questions/63986278/vue-3-v-deep-usage-as-a-combinator-has-been-deprecated-use-v-deepinner-se

**Review by ymarcon — CHANGES_REQUESTED** [review]
Tested, works very well, except some minor comments (+ favicon is vuetify's one); also you should run `npm audit fix`.

File: frontend/src/components/InfoMarkdown.md (hunk 1 above)
Comment (pierreleripoll) [reply frontend/src/components/InfoMarkdown.md]: done

**Comment by pierreleripoll** [comment]
@ymarcon  thanks for feedback ! I modified the app according to them, let me know if this is ok for you

**Review by ymarcon — APPROVED** [review]
LGTM :+1:

**Pull request merged**